midend/parserUnroll.cpp

```diff
@@ -172,6 +173,12 @@ class ParserStateRewriter : public Transform {
         ExpressionEvaluator ev(refMap, typeMap, valueMap);
         auto *value = ev.evaluate(expression->right, false);
         if (!value->is<SymbolicInteger>()) return expression;
+        if (!value->to<SymbolicInteger>()->isKnown()) {
+            ::warning(ErrorType::ERR_INVALID,
+                      "ParserUnroll can't concretize uninitialized value:\n%1%", expression->right);
```

Comment:
I think this error message could be more user-friendly.
Most users won't know what 'concretize' means here.

Reply:
Perhaps "uninitialized value prevents loop unrolling". That's not great, but it's a bit better.